src/secondAssignment_14March2023/FindDestinationCity.java

```diff
@@ -0,0 +1,33 @@
+package secondAssignment_14March2023;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class FindDestinationCity {
+    public void destinationCity(String[][] input) {
+        ArrayList<String> city = new ArrayList<>();
+        for (int row = 0; row < input.length; row++) {
+            for (int column = 0; column < 2; column++) {
```

Comment:
why 2? it should be dynamic

Reply:
Done!! Added code to calculate length of column.